tests/unit_tests/training/test_run_recipe_qwen3_omni.py

```
@@ -0,0 +1,159 @@
+        "recipe_module": qwen3_omni_recipe_module,
+    }
+    return module, test_handles
+
+
+class TestRunRecipeQwen3Omni:
+    """Tests for wiring Qwen3-Omni into the generic training entrypoint."""
+
+    def test_load_forward_step_returns_qwen3_omni_handler(self):
+        """The run_recipe registry should expose qwen3_omni_step."""
+
+        module, handles = _load_run_recipe_module()
```

Ruff (0.15.9)

[warning] Unpacked variable `handles` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)